src/services/user/model.js

```diff
@@ -25,7 +27,7 @@ const userSchema = new Schema({
 
 	birthday: { type: Date },
 
-	importHash: { type: String },
+	importHash: { type: String, index: true },
```

Comment:
how often is this used?

Reply:
index would be small I guess, but if we don't have we would run in a collscan here

Reply:
It is only for new temp user that are created by teachers, or admins.